lib/cdo/workshop_constants.rb

```diff
@@ -0,0 +1,35 @@
+module WorkshopConstants
```

Comment:
This would also fail with a test of just rendering that page (since it
would fail on ActivityConstants::PROGRAM_TYPE being undefined).
Unfortuntely AFAIK pegasus is not set up well for that...

On Tue, Nov 17, 2015 at 8:30 PM, Andrew Oberhardt <notifications@github.com>
wrote:

> In lib/cdo/workshop_constants.rb
> <https://github.com/code-dot-org/code-dot-org/pull/5472#discussion_r45158944>
> :
>
> > @@ -0,0 +1,35 @@
> > +module WorkshopConstants
>
> It is used here:
> https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/sites.v3/code.org/public/ops-dashboard/index.haml#L73-L94
>
> :( I didn't turn this up in my earlier search and it didn't fail any rake
> tests because it's in the javascript. Do we have tests for that?
>
> So, I'll keep it in shared and update ops_dashboard to use
> WorkshopConstants
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/code-dot-org/code-dot-org/pull/5472/files#r45158944>.
>